src/proteus/dswx_hls.py

```diff
@@ -2539,44 +2930,44 @@ def parse_runconfig_file(user_runconfig_file = None, args = None):
     else:
         runconfig = default_runconfig
 
-    runconfig_constants = RunConfigConstants()
+    runcfg_consts = RunConfigConstants()
```

Comment:
BTW, I'd prefer to keep `runconfig_constants` rather than `runcfg_consts`. I think the first former is easier to read, and the difference in variable name length is not that big to justify the change. Would you be OK with keeping it as `runconfig_constants`?

Reply:
Either way is fine with me. The variable name was shortened in response to the PR comment (https://github.com/nasa/PROTEUS/pull/33#discussion_r1039012166) asking lines remain under 80 characters. By using shorter variable names, it made it easier to keep several lines under that limit.

Would you prefer to have a longer variable name, and more lines split onto two lines?

Reply:
Per our offline discussion, the variable name was changed back to `runconfig_constants ` from from `runcfg_consts`.